packages/config/src/memory/__tests__/memory.config.test.ts

```diff
@@ -176,7 +175,9 @@ o.spec('MemoryConfig', () => {
   });
 
   o('The latest imagery should overwrite the old ones', async () => {
+    await new Promise((resolve) => setTimeout(resolve, 5));
     const idLater = ulid();
+    await new Promise((resolve) => setTimeout(resolve, 5));
```

Comment:
There are async implementations of `setTimeout` in node https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options

```typescript 
import timers from 'node:timers/promises'
await timers.setTimeout(5)
```